learn/get-started/write-a-graphql-api-with-ballerina.md

```diff
@@ -46,9 +46,9 @@ The `Query` type has two fields.
 The `Mutation` type has a single field.
 * The `add` field - This field will add a given entry to the data source.
 
-## Create a datasource for the project
+## Create a datasource for the package
```

Comment:
Let's keep project usage here. I don't think we should change it.

Reply:
Updated in https://github.com/ballerina-platform/ballerina-dev-website/pull/4524/commits/848fe2d2779e36676e21b6d44eebf5c17f076c96.